tempesta_fw/http_parser.c

```diff
@@ -905,14 +905,14 @@ __parse_connection(TfwHttpMsg *hm, unsigned char *data, size_t len)
 
 	__FSM_STATE(I_Conn) {
 		TRY_STR_LAMBDA("close", {
-			if (msg->flags & TFW_HTTP_CONN_KA)
+			if (tfw_http_msg_conn(msg))
 				return CSTR_NEQ;
-			msg->flags |= TFW_HTTP_CONN_CLOSE;
+			tfw_http_msg_set_conn(msg, TFW_HTTP_CONN_CLOSE);
```

Comment:
I don't like the patch. `tfw_http_msg_conn(msg)` doesn't equal to `msg->flags & TFW_HTTP_CONN_KA` and `tfw_http_msg_set_conn(msg, TFW_HTTP_CONN_CLOSE)` does more operations than `msg->flags |= TFW_HTTP_CONN_CLOSE`. Probably in this particular case the statements are equal  and performance penalty is invisible, however the change just makes the code bit nicer (meantime `msg->flags & TFW_HTTP_CONN_KA` says more for me than `fw_http_msg_conn(msg)`) and this `bit nicer` doesn't outweight the badnesses.